### DISCUSSION TITLE
联机什么时候出啊

### DISCUSSION THREAD

**Discussion by ikun312** [Meloong-Git/PCL#3245] [提问] [answered]
我都烦死了只能自己玩

**Comment by allMagicNB** [comment]
# 你去用BakaXL吧

**Comment by iPodtouch6557** [comment]
等就行了
先用隔壁启动器凑合着

**Comment by WForst-Breeze** [comment, marked as answer]
猫曰：至于联机，因为网易还在继续代理中国版，我和 HMCL 的维护者都觉得还得谨慎一点……况且现在也有 Mod 支持联机了……咳咳。

在此之前可以用内网穿透或相关模组。总之别急

**Comment by allMagicNB** [comment]
总之
# 我知道你很急，但是你先别急
**急急复急急**——TT702

**Comment by Deep-Dark-Forest** [comment]
让我来打个广告，但是真心好用
【免费低延迟联机工具 支持帕鲁|MC|泰拉瑞亚.....-哔哩哔哩】 https://b23.tv/mjAgpHu
